src/js/site/main.js

```diff
@@ -402,3 +403,40 @@ $(document).ready(function () {
 
 // Currently this function is only used for the insights table
 $(document).ready(highlightURLHash());
```

Comment:
```suggestion
$(highlightURLHash);
```
nit but can you try this without the function invocation? i believe it should work and not give a vscode warning.